### PR TITLE
rename LR type to Requestor Stock Dispatch

### PR DESCRIPTION
**Pull request by lepistone**
Which is more functionally sound.

### CODE REVIEW

**Comment by gurneyalex** [comment]
I'd expect to see a change in the .pot / .po files

**Comment by lepistone** [comment]
@gurneyalex done

File: logistic_requisition_donation/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: logistic_requisition_donation
+#: model:ir.model,name:logistic_requisition_donation.model_logistic_requisition_cost_estimate
+msgid "Create cost estimate of logistic requisition lines"
+msgstr "Create cost estimate of logistic requisition lines"
```
Comment (yvaucher) [inline logistic_requisition_donation/i18n/fr.po]: Please leave fr translation empty when not translated.

**Comment by yvaucher** [comment]
Appart the wrong fr translations values LGTM :+1:

File: logistic_requisition_donation/i18n/fr.po (hunk 1 above)
Comment (lepistone) [inline logistic_requisition_donation/i18n/fr.po]: @yvaucher this is generated with `--i18n-export`. I think it is correct for the `pot` file to have empty strings, and for the `fr` file to have untranslated ones. That is what we see in the interface, in fact.

Comment (yvaucher) [inline logistic_requisition_donation/i18n/fr.po]: @lepistone The issue there is that those strings will be seen as translated but they aren't. Am I wrong?

**Comment by lepistone** [comment]
thanks @yvaucher. `fr.po` is now empty.

**Pull request merged**